pyro/contrib/epidemiology/models.py

```diff
@@ -699,6 +699,7 @@ def transition(self, params, state, t):
         mask_t = self.mask[t] if t_is_observed else False
         data_t = self.data[t] if t_is_observed else None
         pyro.sample("obs_{}".format(t),
+                    # FIXME Delta is incompatible with relaxed inference.
```

Comment:
should this be an assert?

Reply:
I was trying to avoid polluting model-side code with inference logic. I may remove this model before release.